src/angularjs/src/app/places/map/places-map.directive.js

```diff
@@ -0,0 +1,121 @@
+(function() {
+
+    /* @ngInject */
+    function PlacesMapController($filter, $http, $sanitize, $scope) {
+        var ctl = this;
+        ctl.map = null;
+        ctl.layerControl = null;
+
+        ctl.$onInit = function () {
+            ctl.mapOptions = {
+                scrollWheelZoom: true
+            };
+
+            // TODO: set center and zoom level by zooming to fit geojson polygon bounds
```

Comment:
My idea is to fix this up when adding pull of neighborhood bounds to zoom to fit that single area, instead of trying to zoom to fit all 13 results bounds.